### DISCUSSION TITLE
SQFlite and Flutter Hot Reload

### DISCUSSION THREAD

**Discussion by mohamad-elbohsaly** [tekartik/sqflite#567] [Q&A]
Certain transactions **only** upon I make a hot reload. Why?

**SQFLite 1.3.2+1 used
Flutter stable version
Dart 2.10**

**Comment by alextekartik** [comment]
That should not be the case. It would be great it you could provide a way to reproduce the issue as this is not something I can start trying to reproduce and fix without having more information.
